docs/scos/dev/front-end-development/migration-guide-upgrade-to-webpack-v5.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Migration guide - Upgrade to Webpack v5
+description: Use the guide to update versions of the Webpack and related modules.
+template: module-migration-guide-template
+---
+
+## Upgrading from version 4.* to version 5.*
+
+This document provides instructions for upgrading Webpack to version 5 in your Spryker project.
+
+## Overview
+
+The Webpack v5 was released 2 years ago.
+So we need to migrate as well to unblock future upgrades of other dependencies.
+
+*Estimated migration time: 1h*
+
+## 1) Update dependencies
+
+{% info_block infoBox "Note" %}
+
+To ensure that dependencies will install correctly, delete `node_modules` and `package-lock.json` before installation 
+using the next command `rm -rf node_modules && rm -rf package-lock.json`.
+
+{% endinfo_block %}
+
+1. Update the following dependencies to the new version:
+
+```json
+{
+    // update
+    "@types/webpack": "~5.28.0", // If using Merchant Portal
+    "clean-webpack-plugin": "~4.0.0",
+    "compression-webpack-plugin": "~10.0.0",
+    "copy-webpack-plugin": "~11.0.0",
+    "mini-css-extract-plugin": "~2.7.2",
+    "optimize-css-assets-webpack-plugin": "~6.0.1",
+    "terser-webpack-plugin": "~5.3.6",
+    "webpack": "~5.74.0",
+    "webpack-merge": "~5.8.0",
```

Comment:
Should we mention the `fast-glob`, `prettier`, `core-js` updates here or it's needed for the angular update?

Reply:
It's not needed even for angular, just was updated